src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,25 @@
+while (free W) {
+	move W
+}
+
+while (free N) {
+	move N
+}
+
+ while (free E) {
+	while (free S) {
```

Comment:
"How nice is their implementation?" -- I think it's intuitive! However, I think there are some parentheses that might not be necessary. In my opinion, parentheses are used to organize pieces of information that might otherwise be difficult to understand. However, for syntax like WhileOuter (free W), I think it would be less work, and just as clear for the user, if he/she would be able to just say "While f W" (I also abbreviated free).